### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by karol-bisztyga**
## Description

Added test for `scrollTo`
Added test-suite for `processColor`
Did a little clean up in the `test-suite`.

### CODE REVIEW

File: Example/test-suite/tests/Colors.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  t.describe('colors', () => {
+    // test basic colors
+    t.it('process red color', () => {
+      const padding = 2;
+      // test red
+      const redStr = processColor('red')
+        .toString(16)
+        .substring(padding + 0, padding + 2);
+      const redNum = Number.parseInt(redStr, 16);
+      t.expect(redNum).not.toBe(0);
```
Comment (Szymon20000) [inline Example/test-suite/tests/Colors.js:16]: Can we check exactly if it's the correct value?

Comment (karol-bisztyga) [reply Example/test-suite/tests/Colors.js]: fixed 7a5999d

File: Example/test-suite/tests/Colors.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+import { processColor } from 'react-native-reanimated';
+
+export const name = 'Colors';
+
+export function test(t) {
+  t.describe('colors', () => {
+    t.it('process basic colors', () => {
+      const testData = [
+        ['red', 'ff0000'],
```
Comment (Szymon20000) [inline Example/test-suite/tests/Colors.js:10]: Does this work? I guess the values should be different for iOS and Android.

Comment (karol-bisztyga) [reply Example/test-suite/tests/Colors.js]: cdf71f0

**Comment by karol-bisztyga** [comment]
The color test could be also run on UI thread

**Comment by Szymon20000** [comment]
> The color test could be also run on UI thread

It would be nice to test is on both UI and JS thread. To test it on UI thread we could pass already computed values from JSThread

**Comment by karol-bisztyga** [comment]
> > The color test could be also run on UI thread
> 
> It would be nice to test is on both UI and JS thread. To test it on UI thread we could pass already computed values from JSThread

It is now done on both UI and JS

File: Example/test-suite/tests/Colors.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        'cyan',
+        'orange',
+        'yellow',
+        'purple',
+      ];
+
+      const uiResults = {};
+      testData.forEach((color) => {
+        // run on JS thread
+        t.expect(processColor(color)).toBe(reaProcessColor(color));
```
Comment (Szymon20000) [inline Example/test-suite/tests/Colors.js:29]: Does it make a any difference if we swap processColor with reaProcessColor?

**Review by Szymon20000 — APPROVED** [review]

File: Example/test-suite/tests/Colors.js (hunk 3 above)
Comment (karol-bisztyga) [reply Example/test-suite/tests/Colors.js:29]: Not at all. The only difference would probably be an order of how they're printed once the test fails, like `test failed: got aaa, expected bbb`

**Pull request merged**